firefox-ios/Client.xcodeproj/project.pbxproj

```diff
@@ -751,16 +751,6 @@
 		8A3233FE28627446003E1C33 /* LocalDesktopFolder.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8A3233FD28627446003E1C33 /* LocalDesktopFolder.swift */; };
 		8A32DD5028B419B300D57C60 /* HomepageMessageCardViewModelTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8A32DD4F28B419B300D57C60 /* HomepageMessageCardViewModelTests.swift */; };
 		8A33221F27DFE318008F809E /* TopSitesDataAdaptorTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8A33221E27DFE318008F809E /* TopSitesDataAdaptorTests.swift */; };
-		8A3345612BA499B7008C52AB /* disconnect-block-fingerprinting.json in Resources */ = {isa = PBXBuildFile; fileRef = 8A3345572BA499B6008C52AB /* disconnect-block-fingerprinting.json */; };
-		8A3345622BA499B7008C52AB /* disconnect-block-advertising.json in Resources */ = {isa = PBXBuildFile; fileRef = 8A3345582BA499B6008C52AB /* disconnect-block-advertising.json */; };
-		8A3345632BA499B7008C52AB /* disconnect-block-cookies-content.json in Resources */ = {isa = PBXBuildFile; fileRef = 8A3345592BA499B6008C52AB /* disconnect-block-cookies-content.json */; };
```

Comment:
I am wondering why do you need to change these?

Reply:
I got those same files modified in some of my branches 😬 Not sure why, maybe there's a step I am missing on my local device

Reply:
I'm not 100% clear on the reason for the change but currently we do need these in the project, the same way the `RemotePasswordRules.json` is added (AFAIU, at least for now we're bundling the default Remote Settings JSON files into the client until we get to "phase 2" of the RS updates).

It might be that Xcode 16 is removing these references automatically for some reason but I'm not sure why. Is anyone encountering build errors related to this? 🤔

Reply:
These are getting removed as they are unused duplicate references. The used references start roughly at lines 122 and 2567 of the xcproj.

Reply:
Yup, made sure its not an XCode issue and is indeed some dupes being removed.

Reply:
Ok perfect thanks for the update everyone, than i'm going to merge this 😃